prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudication-container/adjudication-container.component.ts

```diff
@@ -488,7 +494,17 @@ export class AdjudicationContainerComponent implements OnInit {
       });
   }
 
-  private getEnrollees({ textSearch, status }: { textSearch?: string, status?: StatusFilterEnum }) {
+  private getEnrollees({ status, ...rest }: {
+    status?: StatusFilterEnum,
+    textSearch?: string,
+    page?: number,
+    sortOrder?: string,
+    assignedTo?: number,
+    appliedDateStart?: string,
+    appliedDateEnd?: string,
+    renewalDateStart?: string,
+    renewalDateEnd?: string
+  }) {
```

Comment:
Should this indicate the return type?

Reply:
oh i guess i changed a different one not this one!